src/org/usfirst/frc4904/robot/commands/GearSet.java

```diff
@@ -0,0 +1,46 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.GearIO;
+import org.usfirst.frc4904.robot.subsystems.GearIO.GearState;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class GearSet extends Command {
+	GearState state;
+	
+	public GearSet(GearIO.GearState state) {
+		requires(RobotMap.Component.gearIntakeOuttake); // Use requires() here to declare subsystem dependencies
+		this.state = state;
+	}
+	
+	// Called just before this Command runs the first time
```

Comment:
Can you remove the unused methods like `initialize` and `end` and `interrupted` ?